### PR TITLE
Integrating storing standup api

### PR DESCRIPTION
**Pull request by ayushdevelop**
# Related Issue

- https://github.com/code-mancers/g-bot/issues/100

# functionalities Added or Changed

- Integrated the api-endpoint to create standup from fronted
- on submitting the data from frontend the api will interact with backend that has `PrismaClient` 
- `PrismaClient` will interact with the database and will store the standup data provided as request params through api

# Additional Info

- React-Admin's `dataProvider` is used to interact with api endpoint

### CODE REVIEW

File: client/src/app.tsx [hunk 1]
```diff
@@ -2,10 +2,9 @@ import { Admin, Resource } from "react-admin";
 import StandupList from "./components/Standups/StandupList";
 import StandupCreate from "./components/Standups/StandupCreate";
 import StandupEdit from "./components/Standups/StandupEdit";
-import fakeDataProvider from "ra-data-fakerest";
-import data from "./db";
+import simpleRestProvider from "ra-data-simple-rest";
 
-const dataProvider = fakeDataProvider(data);
+const dataProvider = simpleRestProvider(`${process.env.REACT_APP_PUBLIC_URL}`);
```
Comment (SujayPrabhu96) [inline client/src/app.tsx:7]:
```suggestion
const dataProvider = simpleRestProvider(process.env.REACT_APP_PUBLIC_URL);
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: client/src/app.tsx (hunk 1 above)
Comment (ayushdevelop) [reply client/src/app.tsx:7]: this has to be a string value, on passing it as a variable it throws an error

File: client/src/app.tsx [hunk 2]
```diff
@@ -2,10 +2,9 @@ import { Admin, Resource } from "react-admin";
 import StandupList from "./components/Standups/StandupList";
 import StandupCreate from "./components/Standups/StandupCreate";
 import StandupEdit from "./components/Standups/StandupEdit";
-import fakeDataProvider from "ra-data-fakerest";
```
Comment (SujayPrabhu96) [inline client/src/app.tsx:5]: If this package is not required anymore, uninstall the package

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: client/src/app.tsx (hunk 2 above)
Comment (ayushdevelop) [reply client/src/app.tsx:5]: okay

**Review by SujayPrabhu96 — APPROVED** [review]

**Pull request merged**